tests/test_packages.py

```diff
@@ -74,6 +74,7 @@
     "MSX-DOS2",
     "MSX-DOS",
     "SCC",
+    "OPL4",
     "ROM",
```

Comment:
Although OPL4 makes total sense as requirement, before adding i, I need to create a new logo for it. You can see all the current requirements with their logos here: https://msxhub.com/list_requirements

Please give me some time to prepare the logo before merging this PR.

Reply:
Sure, i will hold it until the logo is there